test/js/bun/shell/commands/exit.test.ts

```diff
@@ -5,6 +5,7 @@ const TestBuilder = createTestBuilder(import.meta.path);
 import { sortedShellOutput } from "../util";
 import { join } from "path";
 
+$.nothrow();
```

Comment:
why? shouldn't the TestBuilder suppress the exception if it's expected to have a non-zero exit code?